src/js/tracks/text-track-display.js

```diff
@@ -294,6 +316,36 @@ class TextTrackDisplay extends Component {
     }
   }
 
+  /**
+   * Updates the displayed TextTrack to be sure it overlays the video when a either
+   * a {@link Player#texttrackchange} or a {@link Player#fullscreenchange} is fired.
+   */
+  updateDisplayOverlay() {
+    if (!this.player_.videoHeight()) {
+      return;
+    }
+
+    const playerWidth = this.player_.currentWidth();
+    const playerHeight = this.player_.currentHeight();
+    const playerAspectRatio = playerWidth / playerHeight;
+    const videoAspectRatio = this.player_.videoWidth() / this.player_.videoHeight();
+    let leftRight = 0;
+    let topBottom = 0;
+
+    if (Math.abs(playerAspectRatio - videoAspectRatio) > 0.01) {
+      if (playerAspectRatio > videoAspectRatio) {
+        leftRight = Math.round((playerWidth - playerHeight * videoAspectRatio) / 2);
+      } else {
+        topBottom = Math.round((playerHeight - playerWidth / videoAspectRatio) / 2);
+      }
+    }
+
+    tryUpdateStyle(this.el_, 'top', getCSSPositionValue(topBottom));
+    tryUpdateStyle(this.el_, 'bottom', getCSSPositionValue(topBottom));
+    tryUpdateStyle(this.el_, 'left', getCSSPositionValue(leftRight));
+    tryUpdateStyle(this.el_, 'right', getCSSPositionValue(leftRight));
```

Comment:
can use `insetInline` instead of `left` and `right`.

In CSS, it's:
```
.vjs-text-track-display: {
  inset-inline: 5px;
}
```

Reply:
I guess the `leftRight` var can also be renamed to match.

Reply:
Thanks, It is a great idea. I have already changed that.